frontend/sass/evictionfree/styles.scss

```diff
@@ -59,3 +59,7 @@ $jf-navbar-content: $black;
   font-family: $jf-title-family;
   letter-spacing: 0;
 }
+
+.jf-nowrap {
+  white-space: nowrap;
+}
```

Comment:
@toolness do we have any utility class like this set up anywhere? Seems useful to me— if not maybe I should put this bit of css somewhere more central?

Reply:
oh sure bud.

Reply:
Maybe in `frontend/sass/_helpers.scss`?